src/internal/storage/chunk/reader.go

```diff
@@ -126,14 +129,43 @@ func (dr *DataReader) DataRef() *DataRef {
 	return dr.dataRef
 }
 
+func (dr *DataReader) Prefetch() error {
+	return dr.Get(io.Discard)
+}
+
 // Get writes the data referenced by the data reference.
 func (dr *DataReader) Get(w io.Writer) error {
-	return Get(dr.ctx, dr.client, dr.memCache, dr.deduper, dr.dataRef.Ref, func(chunk []byte) error {
-		if dr.offset > dr.dataRef.SizeBytes {
-			return errors.Errorf("DataReader.offset cannot be greater than the dataRef size. offset size: %v, dataRef size: %v.", dr.offset, dr.dataRef.SizeBytes)
-		}
+	if dr.offset > dr.dataRef.SizeBytes {
+		return errors.Errorf("DataReader.offset cannot be greater than the dataRef size. offset size: %v, dataRef size: %v.", dr.offset, dr.dataRef.SizeBytes)
+	}
+	ref := dr.dataRef.Ref
+	b := backoff.NewExponentialBackOff()
+	b.InitialInterval = 1 * time.Millisecond
+	cb := func(chunk []byte) error {
```

Comment:
Could we just inline this in the getFromCache call?

Reply:
yeah sure